### ISSUE TITLE
No action is dispatched when using the browser's back and forward buttons

### ISSUE THREAD

**Issue by intojs** [ngrx/router-store#44]
I think it would be a good idea to dispatch actions when we use the browser's back and forward buttons.
This way we can listen to these actions in our reducers and manipulate the state.
Thank you:)

**Comment by artem-v-shamsutdinov** [comment]
Yep, same issue here.  We started a new project here and have our router.navigate calls in a single @Effect that centralizes routing side effects.  But now back and forward browser navigation is broken. :(
